src/main/resources/log4j2.xml

```diff
@@ -25,7 +25,7 @@
 		</RollingFile> -->
     </Appenders>
     <Loggers>
-        <logger name="iudx.resource.server" level="${env:LOG_LEVEL}"
+        <logger name="iudx.resource.server" level="${env:LOG_LEVEL:-DEBUG}"
```

Comment:
Does this suggest a default environment variable?

Reply:
Yes

> Does this suggest a default environment variable?

Yes, default log level is debug, if the env variable LOG_LEVEL is not defined.